1e/README.md

```diff
@@ -0,0 +1,168 @@
+# 1E
+
+## Overview
+
+At [1E][1], we reimagine how technology serves people and create new ways for IT to shape the future of work.
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account.
+
+Integrating Datadog with 1E enables IT administrators to:
+
+- View device data and related information in one place in Datadog
+- View Digital Employee Experience (DEX) score and related information
+- Build a customizable dashboard inside Datadog for customer specific insights.
+
+## Setup
+
+### Prerequisites
+
+To configure your integration between Datadog and 1E you will need to have the following in place:
+
+- 1E 8.4
+  - platform installed, with at least one 1E client connected to provide metrics data
+  - a user that is assigned to the **Full Administrator** role
+  - 1E PowerShell Toolkit 2.0 or later, available from the [1E Support portal][2]
+- Datadog
+  - a user that is assigned to the **Datadog Admin** role and able to download the 1E Dashboard from the Marketplace.
+
+### Configuration
+
+Steps required to configure your integration between Datadog and 1E.
+
+#### Installing the 1E Dashboard
+
+You can install the 1E Dashboard from the Datadog Marketplace at any time in this process.
+
+To install, navigate to **Integrations** in your Datadog instance, search for **1E**, and open **1E Dashboard**.
+
+In 1E Dashboard, click on **Install Integration**.
+
+The dashboard's tiles will show data after you have configured integration with 1E.
```

Comment:
Is there a separate integration for the dashboard? If not, the customer will simply need to: 
1. Go to the Datadog integrations page
2. Search for 1E 
3. Install the integration

This will automatically populate the dashboard

Reply:
Removed this part